### ISSUE TITLE
Lymphocytic infiltration of ...

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6253]
**Preferred term label:**
Lymphocytic infiltration of ... lung, liver

See Fig of https://www.pnas.org/content/94/7/3168.long

analogous to https://hpo.jax.org/app/browse/term/HP:0031322

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
PMID:24116927  OMIM:606367 
Lymphocytic infiltration of the lung  1/1

**Comment by pnrobinson** [comment]
@mgriese is 'Lymphocytic infiltration of the lung' a valid new term or should it be related to an existing term?

**Comment by mgriese** [comment]
Can be a term; is however very broad "lung", as the airways are first affeected and then also the interstitial space.  This is the same as with HP:0032216; that term is associated with a disease. 
In the lung we have the same situtation; this is an old point of discussion between us: there are many terms which are "by principal" COMPLEX phenotypic features, and which have then been used as disease names/diseases (as there was no further clue to name the disease). Now more and more molecular diseases can be attached to the terms and the development oges into the other direction, the terms become again a (complex) term and are associated with disease.
For example in our paper exactly on this term (32040879): we had 10 or so cases with this "diagnosis", 4 or 5 could be defined  molecularly as specific diseases (CTL4, COPA,..), but others not yet. 
So bottom line (again): I recommend to define these complex pattern as terms in addition to diseases, because the term for the main feature alone (lymphocytic infiltration) is not the same a term which also describes the other aspects (e.g. formation of follicles).

**Comment by pnrobinson** [comment]
@mgriese can we discuss this today?

**Comment by pnrobinson** [comment]
Pulmonary interstitial lymphocyte infiltration (upper level grouping term)
-Lymphoid interstitial pneumonia (subclass)
-Follicular bronch.
(etc0

**Comment by pnrobinson** [comment]
added term
